src/bundle/Resources/public/js/scripts/sidebar/main.menu.js

```diff
@@ -38,11 +39,26 @@
 
         doc.removeEventListener('mousemove', collapseSecondLevelMenu);
     };
-    const showSecondLevelMenu = ({ currentTarget }) => {
+    const showAndHideSecondLevelMenu = ({ currentTarget }) => {
```

Comment:
maybe we'll change the name to:
```suggestion
    const toggleSecondLevelMenu = ({ currentTarget }) => {
```

Reply:
`toggleSecondLevelMenu` already exists, but I agree that we can improve the method name